### ISSUE TITLE
Example Results with Pretrained Autoencoder

### ISSUE THREAD

**Issue by Htermotto** [mlmed/torchxrayvision#112]
Hi,

Are there any example results with your pretrained autoencoder? I tried it with CheXpert but its decoded images were mostly gray. Wondering if there is an example and I might be missing some preprocessing? I am using the normalize function provided in the XRV repo to rescale my images.

Example on CheXpert:

<img width="382" alt="image" src="https://user-images.githubusercontent.com/9905056/202750585-f569cc98-79cc-425a-a358-60077738a56a.png">

Thanks!

**Comment by ieee8023** [comment]
I put together a demo notebook here? https://github.com/mlmed/torchxrayvision/blob/master/scripts/autoencoder.ipynb

It seems like something is wrong there. Was the autoencoder loaded with weights? `xrv.autoencoders.ResNetAE(weights="101-elastic")`

**Comment by Htermotto** [comment]
Thanks, sorry for not finding that earlier. I was able to reconfirm the better results with NIH. Now that I look at the autoencoder description more closely, I think it may not be trained on the CheXpert images. Do you know if it was?

**Comment by ieee8023** [comment]
> Thanks, sorry for not finding that earlier.

I just added it after reading your message!

>Do you know if it was?

It was not trained on chexpert. It was trained in NIH PC RSNA and MIMIC

The chexpert images work for me though as shown below. Can you make a small script to share and I can try it?

<img width="800" alt="Screenshot 2022-11-19 at 10 38 48" src="https://user-images.githubusercontent.com/446367/202866421-b787be40-8ead-4cc9-a6ac-9dad56d70c63.png">

**Comment by Htermotto** [comment]
hi, thanks for taking a look! I was able using the example you provided to rerun the encoder on chexpert with much better results. I think I was missing some of the image transformations.
